cmd/aries-agent-mobile/pkg/api/notifier.go

```diff
@@ -0,0 +1,17 @@
+/*
+ Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package api
+
+// Notifier represents a notification dispatcher.
+type Notifier interface {
+
+	// Notify sends the given message to all of the subscribers.
+	Notify(topic string, message []byte) error
+
+	// GetPayload returns the payload message.
+	GetPayload() []byte
```

Comment:
we don’t  need this function

Reply:
I just realized that the best name is `Handle`. Please replace Notify with `Handle`.
The `Handle` function plays a callback function role.

Reply:
And `Notifier` will be `Handler`.

Reply:
Make sense?